truffle.js

```diff
@@ -25,7 +25,12 @@ module.exports = {
     }
   },
   mocha: {
-    reporter: "mocha-circleci-reporter"
+    reporter: "mocha-circleci-reporter",
+    reporterOptions: {
+      currency: "USD",
+      gasPrice: 20,
```

Comment:
Current 'Safe Low' gas price (according to ETH Gas Station) is 3. How about making this 5?